data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+devtools::use_data(google_trends, overwrite = TRUE)
+
+mediacloud_hurricanes <- read_csv("data-raw/puerto-rico-media/mediacloud_hurricanes.csv")
+mediacloud_hurricanes <- clean_names(mediacloud_hurricanes)
+mediacloud_hurricanes$date <- as.Date(mediacloud_hurricanes$date, format= "%m / %d / %y")
+devtools::use_data(mediacloud_hurricanes, overwrite = TRUE)
+
+mediacloud_states <- read_csv("data-raw/puerto-rico-media/mediacloud_states.csv")
+mediacloud_states <- clean_names(mediacloud_states)
+mediacloud_states$date <- as.Date(mediacloud_states$date, format= "%m / %d / %y")
+devtools::use_data(mediacloud_states, overwrite = TRUE)
+
+mediacloud_online_news <- read_csv("data-raw/puerto-rico-media/mediacloud_top_online_news.csv")
+devtools::use_data(mediacloud_top_online_news, overwrite = TRUE)
+
+mediacloud_trump <- read_csv("data-raw/puerto-rico-media/mediacloud_trump.csv")
+mediacloud_trump <- clean_names(mediacloud_trump)
+mediacloud_trump <- mediacloud_trump %>%
+  rename(
+    puerto_rico = title_puerto_rico,
+    puerto_rico_and_trump = title_puerto_rico_and_title_trump_or_title_president,
+    florida = title_florida, 
+    florida_and_trump = title_florida_and_title_trump_or_title_president,
+    texas = title_texas,
+    texas_and_trump = title_texas_and_title_trump_or_title_president
+  )
+devtools::use_data(mediacloud_trump, overwrite = TRUE)
+
+tv_hurricanes <- read_csv("data-raw/puerto-rico-media/tv_hurricanes.csv")
+tv_hurricanes <- clean_names(tv_hurricanes)
+tv_hurricanes$date <- as.Date(tv_hurricanes$date, format= "%m / %d / %y")
+devtools::use_data(tv_hurricanes, overwrite = TRUE)
+
+tv_hurricanes_by_network <- read_csv("data-raw/puerto-rico-media/tv_hurricanes_by_network.csv")
+tv_hurricanes_by_network <- clean_names(tv_hurricanes_by_network)
+tv_hurricanes_by_network <- tv_hurricanes_by_network %>%
+  mutate(
+    date = as.Date(date, format= "%m / %d / %y"),
+    query = as.factor(query)
+  )
+devtools::use_data(tv_hurricanes_by_network, overwrite = TRUE)
+
+tv_states <- read_csv("data-raw/puerto-rico-media/tv_states.csv")
+tv_states <- clean_names(tv_states)
+tv_states$date <- as.Date(tv_states$date, format= "%m / %d / %y")
+devtools::use_data(tv_states, overwrite = TRUE)
+
+# riddler-pick-lowest---------------------------------------------------------------------
+riddler_pick_lowest <- read_csv("data-raw/riddler-pick-lowest/low_numbers.csv")
+riddler_pick_lowest <- clean_names(riddler_pick_lowest)
+devtools::use_data(riddler_pick_lowest, overwrite = TRUE)
+
+# sandy-311-calls---------------------------------------------------------------------
+sandy_311 <- read_csv("data-raw/sandy-311-calls/sandy-311-calls-by-day.csv")
+sandy_311 <- clean_names(sandy_311)
+
+sandy_311 <- sandy_311 %>%
+  rename(
+    nyc_311 = nyc_3_1_1,
+    nyc_service = nycservice,
+    nys_emergency_mg = nysemergencymg
+  ) %>%
+  mutate(
+    date = as.Date(date, format= "%m / %d / %y")
+  )
+sandy_311<-gather(sandy_311, "agency", "num_calls", 2:24)
```

Comment:
oh, I see you already know how to `gather`! Please move this code to the help file; we try not to alter the datasets toooooo much from original form.